src/bTagSF.cc

```diff
@@ -323,10 +326,13 @@ std::vector<float> bTagSF::getEvtWeightShifted (std::vector <std::pair <int, flo
 {
     // Systematics names
     std::vector<std::string> systNames = {
-        "up_jes", "up_lf", "up_hf", "up_hfstats1", "up_hfstats2", "up_lfstats1",
-        "up_lfstats2", "up_cferr1", "up_cferr2",
-        "down_jes", "down_lf", "down_hf", "down_hfstats1", "down_hfstats2", "down_lfstats1",
-        "down_lfstats2", "down_cferr1", "down_cferr2"
+        "up_jes", "up_lf", "up_hf", "up_hfstats1", "up_hfstats2", "up_lfstats1", "up_lfstats2", "up_cferr1", "up_cferr2",
```

Comment:
shouldn't we remove up_jes and down_jes from here?

Reply:
Those are the SFs for the total JES uncertainty, we can save them and not use them.

Reply:
And also avoid to change the sacred order of the splitter uncertainties, fine eheh